test/gpu/native/studies/sort/radixSortGpu.chpl

```diff
@@ -433,33 +48,34 @@ module RadixSort {
     var prefixSum: [0..<buckets] uint;
 
     // Ceiling on number of iterations based on max element in array
+    var maxVal =  max(uint);
     // Reduce can be used to do this faster
+    // We do this because the preferred ways below either don't work
+    // or take too long
     // But I'm leaving this as is due to https://github.com/chapel-lang/chapel/issues/22736
-    var max = inputArr[0];
-    for i in 1..<inputArr.size {
-      if inputArr[i] > max {
-        max = inputArr[i];
-      }
-    }
+    // var maxVal = max reduce inputArr; // Doesn't work actually
+    // var maxVal = inputArr[0]; // Takes toooo long
+    // for i in 1..<inputArr.size {
+    //   if inputArr[i] > maxVal {
+    //     maxVal = inputArr[i];
+    //   }
+    // }
```

Comment:
This comment and the commented code... are the comments still consistent with what's happening here? If so, I am not fully following what's going on.

Reply:
There is a discrepancy here with the comments.

In the new gpu radix sort algorithm, we have the input array already on the GPU, this means serially iterating over the entire array trying to find the max value takes way too long for some reason. And using `max reduce` is something I'm trying to avoid till we have a good reduce story.
So I opted to just use a the absolute maximum ceiling which is `max(uint)`

Now in this case, this is not the new radix sort algorithm, this is the old atomic radix sort. So the array isn't actually on the GPU. Therefore ideally I should be using the `max reduce` syntax here. I can do that if we want, but that is tangential to how well the sorting is actually performing on GPUs. 
Thinking about it more, maybe a fair comparison for the two algorithms would be when they both have the same number of iterations, but that is already not the case since by sorting 16 bits at a time in the atomic sort and 8 bits at a time in the new radix sort, we already do half the number of iterations on the atomic sort.

I guess in that sense I should be using `max reduce` here to give atomic sort the best shot at it's own performance so we can compare it to the new radix sort as it continues to improve. I can also change the comments to say so.